gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/testing/package-info.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * A testing helper for Google Cloud Datastore.
+ *
+ * <p>A simple usage example:
+ * <pre> {@code
+ * LocalGcdHelper gcdHelper = LocalGcdHelper.start(PROJECT_ID, PORT_NUMBER);
```

Comment:
Fine as is but I do wonder if we should suggest where to put it (`@BeforeClass`, `@AfterClass`) or if we don't want to be JUnit specific to separate it to 2 different code segments: before test, after test.